lib/file_download.rb

```diff
@@ -21,18 +18,15 @@ def fetch
 
   private
 
-  def download_file(url, redirect_limit = MAX_REDIRECTS)
```

Comment:
Actually, thinking about it, this doesn't _really_ need to be a method on its own at all now. The logic can just be moved into the `fetch` method above